src/ranged.cpp

```diff
@@ -1259,8 +1259,8 @@ static inline void eject_casing( player& p, item& weap ) {
     // some magazines also eject disintegrating linkages
     const auto mag = weap.magazine_current();
     if( mag && mag->type->magazine->linkage != "NULL" ) {
-        item linkage( mag->type->magazine->linkage, calendar::turn );
-        linkage.charges = 1; // needs charge 1 to stack properly with other linkages
+        // needs charge 1 to stack properly with other linkages
```

Comment:
Is this comment still accurate? Afaik, it needs the explicit charges count because it would otherwise get the default charge count. In any case, the charges count of those items (that are counted by charges) should not affect whether they stack with other items at all.

Similar comment is below for casings.